headers/types/dungeon_mode/dungeon_mode.h

```diff
@@ -240,6 +240,59 @@ struct statuses {
 ASSERT_SIZE(struct statuses, 118);
 #pragma pack(pop)
 
+#pragma pack(push, 1)
```

Comment:
I don't think you need packing here? The struct layout doesn't have any implicit padding.